packages/brokeneck-react/src/components/Navigation.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { screen, render } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom'
+
+import RootContext from '../components/RootContext'
+import ThemeSwitcherProvider from '../components/ThemeSwitcherProvider'
+
+import Navigation from './Navigation'
+
+describe('Navigation', () => {
+  it('should render the Navigation component with dark theme', () => {
+    localStorage.setItem('themeType', 'dark')
+
+    render(
+      <RootContext.Provider value={{ provider: { name: 'cognito' } }}>
+        <ThemeSwitcherProvider>
+          <MemoryRouter initialEntries={['/users']}>
+            <Navigation />
+          </MemoryRouter>
+        </ThemeSwitcherProvider>
+      </RootContext.Provider>
+    )
```

Comment:
can you try to avoid this repetition? the options here are:

- extract a helper method used by the two (or more) tests 
- use the rtl features to wrap the rendered component with the wrapper components https://testing-library.com/docs/react-testing-library/setup#custom-render

or a mix of the two. this applies to other test modules as well, though this one is more visible because the wrapper is large

Reply:
Absolutely, that's one of the list of things to do. I wanted to create more tests to see what kind of wrappers would be needed.
I suspect there will be some common ones to extract in a test-utils library.